package.json

```diff
@@ -501,6 +554,16 @@
     "configuration": {
       "title": "Confluent",
       "properties": {
+        "confluent.chat.maxIterations": {
+          "type": "number",
+          "default": 2,
+          "minimum": 1,
+          "maximum": 2,
```

Comment:
Teeny because otherwise the screenshot below happens. But may not be ideal for other uses. Need to consider best option.
<img width="1302" alt="Screenshot 2025-04-22 at 1 58 40 PM" src="https://github.com/user-attachments/assets/46eba93b-eab2-44b3-b4c3-6156f27c65ed" />